Installer/dolphin-emu-nogui.6

```diff
@@ -0,0 +1,56 @@
+.TH DOLPHIN-EMU-NOGUI 6 "March 4, 2016"
```

Comment:
I can made a commit, where should I stick it? Right in Data? or in a subfolder in Data?